inference/generativeai/llm-workshop/deploy-gptq-quant-tgi/deploy_gptq_quant_tgi.ipynb

```diff
@@ -0,0 +1,816 @@
+{
```

Comment:
<p>Add badges as described in the CONTRIBUTING.md file</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>